src/main/java/org/opentripplanner/api/mapping/RouteMapper.java

```diff
@@ -26,7 +26,7 @@ public static ApiRoute mapToApi(Route domain) {
     api.id = FeedScopedIdMapper.mapToApi(domain.getId());
     api.agency = AgencyMapper.mapToApi(domain.getAgency());
     api.shortName = domain.getShortName();
-    api.longName = domain.getLongName();
+    api.longName = domain.getLongName().toString();
```

Comment:
Here is another place where you have to use the I18nStringMapper.

Reply:
Done